sql-cli/sql_cli/project.py

```diff
@@ -145,8 +173,9 @@ def initialise(self) -> None:
             dirs_exist_ok=True,
         )
         self._initialise_global_config()
-        initialise_airflow(self.airflow_home, self.airflow_dags_folder)
-        self._remove_unnecessary_airflow_files()
+        for env in self.environments_list:
+            self._initialise_env_config(env)
+            self._initialise_airflow_for_environment(env)
```

Comment:
How will we take care of initialising Airflow when a user adds a new environment?

Reply:
I am thinking if we should provide a command to create a new environment which would take care of it.

Reply:
>  How will we take care of initialising Airflow when a user adds a new environment?
@pankajkoti that is being handled in the `load_config`

In future we can look into exposing creating a new environment in a separate command, but it feels we don't need this ATM